eg/Classes/WindowsVersion.py

```diff
@@ -0,0 +1,342 @@
+# any later version.
+#
+# EventGhost is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with EventGhost. If not, see <http://www.gnu.org/licenses/>.
+
+import platform
+from collections import OrderedDict
+
+WINDOWS_VERSIONS = OrderedDict()
+WINDOWS_VERSIONS["XP"] = [[5, 1], [5, 2]]
+WINDOWS_VERSIONS["XP32"] = [5, 1]
+WINDOWS_VERSIONS["XP64"] = [5, 2]
+WINDOWS_VERSIONS["VISTA"] = [6, 0]
+WINDOWS_VERSIONS["7"] = [[6, 1], [6,2]]
+WINDOWS_VERSIONS["8"] = [6, 2]
+WINDOWS_VERSIONS["80"] = [6, 2]
+WINDOWS_VERSIONS["81"] = [6, 3]
+WINDOWS_VERSIONS["10"] = [10, 0]
+
+
+def _compare(opp, other):
+    if isinstance(other, int):
+        other = str(other)
+    else:
+        other = other.upper()
+
+    if other not in WINDOWS_VERSIONS.keys():
+        raise WindowsVersionError(other)
+
+    ver = platform.version().split('.')
+    this = [int(ver[0]), int(ver[1])]
+
+    versions = [WINDOWS_VERSIONS[other]]
+    if opp == '>':
+        return all(this > version for version in versions)
+    elif opp == '<':
+        return all(this < version for version in versions)
+    elif opp == '<=':
+        return all(this <= version for version in versions)
+    elif opp == '>=':
+        return all(this >= version for version in versions)
+    elif opp == '==':
+        return all(this == version for version in versions)
+    elif opp == '!=':
+        return all(this != version for version in versions)
+
+
+class WindowsVersionError(Exception):
+    def __init__(self, msg):
+        self.msg = msg
+
+    def __str__(self):
+        return (
+            "'{0}' is not in the list of supported values:\n"
+            "\t{1}\n".format(self.msg, ", ".join(WINDOWS_VERSIONS.keys()))
+        )
+
+
+class WindowsVersion:
+    """
+    A convenience module for checking the windows version.
+
+    In addition to the *IsXY()* methods, you can use comparison like this:
+
+    ``eg.WindowsVersion() OPERATOR "KEY"``
+
+    ``OPERATOR`` is one of ``<``, ``<=``, ``==``, ``!=``, ``>=``, ``>``
+
+    +-----------+------------------------------------+
+    | ``KEY``   | Windows version(s)                 |
+    +===========+====================================+
+    | ``XP``    | *all of XP32 and XP64*             |
+    +-----------+------------------------------------+
+    | ``XP32``  | Windows XP,                        |
+    |           | Windows Tablet PC,                 |
+    |           | Windows Media Center Edition 2002, |
+    |           | Windows Media Center Edition 2004, |
+    |           | Windows Media Center Edition 2005  |
+    +-----------+------------------------------------+
+    | ``XP64``  | Windows Server 2003,               |
```

Comment:
you need to add a line in here

    |           | Windows XP Professional                  |